weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/Links.tsx

```diff
@@ -176,6 +176,28 @@ export const OpVersionLink: React.FC<{
   );
 };
 
+export const OpCallLink: React.FC<{
```

Comment:
Would it be possible to modify the `CallLink` below? It would be ideal to have a single CallLink component that has common formatting.

Reply:
It wasn't clear to me if we also wanted this change to affect the ObjectVersionPage.

Reply:
I think it would be good to have consistency in rendering of the same concept as much as possible